src/connection-manager/update-amendments-from-json.ts

```diff
@@ -0,0 +1,63 @@
+import 'dotenv/config'
+import * as fs from 'fs'
+
+import { rippleTimeToUnixTime } from 'xrpl'
+
+import { query } from '../shared/database/utils'
+import { AmendmentEnabled } from '../shared/types'
+import logger from '../shared/utils/logger'
+
+const log = logger({ name: 'database-agreement' })
+const filePath = 'src/shared/data/amendments_enabled.json'
+
+interface AmendmentEnabledJson {
+  networks: string
```

Comment:
It should not be called networks because each are different.

Reply:
It is called `networks` across other tables as well even though all should be a 1-1 match. I have created an issue to fix them:
https://github.com/ripple/validator-history-service/issues/165